README.md

```diff
@@ -117,69 +138,9 @@ Once installed, pre-commit can be run with:
 pre-commit run --all-files
 ```
 
-### UiPlugins
-
-#### Adding a ui-plugin
-
-1. Create a folder in `src/plugins`. This will be your plugin, and must have a `index.js`-file that has a default export
-   of the type `export const plugins: TPlugin[]`
-2. Add this plugin to the list of plugins to load in `src/plugins.js`
-3. Done!
-
-#### Using the ui-plugin
-
-Associate a blueprint type with a set of uiRecipes that uses the ui-plugin
-
-1. Create a `CORE:RecipeLink`-entity (see `app/data/DemoApplicationDataSource/instances/recipe_links/demoApp.json` as an
-   example)
-2. Create a _lookup_ with the app name and the _RecipeLinks_ for it (DMSS must be running on localhost:5000)
-   - `dm create-lookup DemoApp DemoApplicationDataSource/instances/recipe_links`
-
-### Using the Tree component
-
-To use the Tree component from [dm-core](https://github.com/equinor/dm-core-packages), visible data sources must be
-specified. If you want to change the datasource name or add new data sources, the visibleDataSources list defined in
-your App.tsx needs to be updated. In the DemoApp,
-[App.tsx](https://github.com/equinor/create-dm-app/blob/main/src/plugins/DemoApp/App.tsx) only has one visible data
-source, DemoApplicationDataSource.
-
-### Link core
+### Link dm-core-packages
 
 If you want to work on the core and don't want to release new core versions to see the changes in create-dm-app, then
 change the `@development-framework/dm-core` from inside `package.json` to point to core locally
-like `link:./../data-modelling-tool/web/packages/dmt-core/`. You have to run `yarn rollup` inside dm-core to get changes,
-since this will bild a new dist that will be picked-up by create-dm-app.
-
-### Connecting to DMSS
-
-If you run the [DMSS API](https://github.com/equinor/data-modelling-storage-service) locally, it will by default be
-available at localhost:8000. If this is the case, you must update an environment variable. This is achieved by editing
-the start script in package.json:
-
-```json
-"start": "REACT_APP_DMSS_URL=http://localhost:5000 react-app-rewired start"
-```
-
-### Resetting data sources
+like `link:./../data-modelling-tool/web/packages/dmt-core/`. You have to run `yarn rollup` inside dm-core to get changes, since this will build a new dist that will be picked-up by create-dm-app.
```

Comment:
Does this work?
```suggestion
like `link:./../dm-core-packages/packages/dm-core/`. You have to run `yarn rollup` inside dm-core to get changes, since this will build a new dist that will be picked-up by create-dm-app.
```